scripts/apply_withdrawals.js

```diff
@@ -14,16 +14,13 @@ module.exports = async (callback) => {
     const state_index = (await instance.stateIndex.call()).toNumber()
     const curr_state = await instance.stateRoots.call(state_index)
 
-    const withdraw_state = await instance.pendingWithdraws.call(slot)
-    if (withdraw_state.appliedAccountStateIndex != 0) {
+    if (await instance.hasWithdrawBeenApplied(slot)) {
```

Comment:
Are we just trying to make use of the public view functions for connivence? OH!!! Is this because this values are now part of a library and can no longer be accessed directly.

Reply:
Exactly. Calling it the old way leads to "method not found" error